packages/api/cms-api/src/generator/crud-generator.decorator.ts

```diff
@@ -71,3 +71,13 @@ export function hasFieldFeature(metadataClass: any, propName: string, option: ke
     const defaultValue = option == "dedicatedResolverArg" ? false : true;
     return crudField[option] ?? defaultValue;
 }
+
+export interface CrudPositionFieldOptions {
+    positionGroupFields?: string[];
```

Comment:
Optional: position is redundant here, maybe just `groupFields` or `fields`?

Reply:
hmm... I called a group of entities with related positions positionGroup. but obviously this will be mistaken in the decorator. I think groupFields or fields will require some explanation as well...

how about one of these: positionSetFields, uniqueByFields, groupByFields? if it were possible to change type from `string[]` to `keyof field[]` fields-suffix could be removed...

Reply:
> if it were possible to change type from string[] to keyof field[] fields-suffix could be removed...

This would probably be possible by making the decorator and the options generic, but is this necessary?

@nsams do you have a suggestion for a name?

Reply:
`groupByFields` sounds good